please_copy_and_adapt/training_parameters.yaml

```diff
@@ -0,0 +1,132 @@
+#       an empty value (all are yaml's equivalent to python's None).
+# Tip:  To set a boolean in yaml, write either True/False or On/Off. Both
+#       choices correspond to python's True/False.
+##################
+
+logging:
+  # level: one of ['error', 'warning', 'info', 'debug']
+  #     Logging level. If null, default = warning.
+  level: warning
+
+experiment:
+  # name: str
+  #     If given, name for the experiment. Else, model will decide the name to
+  #     give.
+
+dataset:
+  # hdf5_filename: str
+  #     Path to the .hdf5 dataset. Should contain both your training subjects
+  #     and validation subjects.
+  hdf5_filename: my_file.hdf5
+  # training_subjs_filename: str
+  #     Txt file containing the list of subjects used for training. One
+  #     subject per line.
+  training_subjs_filename: training_subjs.txt
+  # validation_subjs_filename: str
+  #     Txt file containing the list of subjects used for validation. One
+  #     subject per line.
+  validation_subjs_filename: validation_subjs.txt
+
+preprocessing:
+  # step_size: float
+  #     Resample all streamlines to this step size (in mm). If null, train on
+  #     streamlines as they are (e.g. compressed).
+  step_size: null
+
+data_augmentation:
+  # add_noise: bool
+  #     Add random gaussian noise to streamline coordinates on-the-fly. Noise
+  #     variance is 0.1 * step-size, or 0.1mm if no step size is used.
+  # toDO!! Add variable for variance
+  add_noise: False
+  # split_ratio: float
+  #     Split a percentage of streamlines at a random point, in each batch. Null
+  #     is equivalent to 0.
+  split_ratio: null
+
+input:
+  neighborhood:
+    # sphere_radius: float
+    #     If not null, a neighborhood will be added to the input information.
+    #     This neighborhood definition lies on a sphere. It will be a list of 6
+    #     positions (up, down, left, right, behind, in front) at exactly given
+    #     radius around each point of the streamlines.
+    #     * Can't be used together with grid_radius.
+    sphere_radius: null
+    # grid_radius: int
+    #     If not null, a neighborhood will be added to the input information.
+    #     This neighborhood definition uses a list of points similar to the
+    #     original voxel grid around each point of the streamlines. Ex: with
+    #     radius 1, that's 27 points. With radius 2, that's 125 points.
+    #     * Can't be used together with sphere_radius.
+    grid_radius: null
+  # add_x_previous_dirs: int
+  #     Concatenate X previous streamline directions to the input vector. Null
+  #     is equivalent to 0.
+  add_x_previous_dirs: null
+
+model:
+  # Add your model's parameters
+
+epochs:
+  # max_epochs: int
+  #     Maximum number of epochs. Suggestion: 100.
+  max_epochs: null
+  # patience: int
+  #     Use early stopping. Defines the number of epochs after which the
+  #     model should stop if the loss hasn't improved. Suggestion: 20.
+  patience: null
+  batch:
```

Comment:
To me the `batch` group should not be grouped under the `epochs` group. I would also group these parameters under some sort of `training` or `learning` group